app/kubernetes-ingress-controller/2.0.x/guides/getting-started-istio.md

```diff
@@ -2,147 +2,460 @@
-✔ Istio operator installed                                                                                                                                                                                                                                    
-✔ Installation complete
+$ export PATH="$PATH:$PWD/istio-1.11.2/bin"
 ```
 
-### Deploy Istio using Operator
+Verify that `istioctl` is working, and run some validation checks on your
+Kubernetes cluster to ensure Istio will deploy to it properly:
 
-Deploy Istio using Istio Operator:
+{% navtabs %}
+{% navtab Command %}
 
 ```console
-$ kubectl create namespace istio-system
-namespace/istio-system created
+$ istioctl x precheck
 ```
+
+{% endnavtab %}
+{% navtab Response %}
+
+```console
+✔ No issues found when checking the cluster. Istio is safe to install or upgrade!
+  To get started, check out https://istio.io/latest/docs/setup/getting-started/
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Now you're ready to move on to Istio deployment.
+
+> **NOTE** If you're interested in other installation methods see the
+  [Istio FAQ][install-method-faq] for more details on the differences
+  between methods.
+
+> **NOTE** If you use one of the other installation mechanisms note that the
+  examples in this guide may not work without experiementation and manual
+  intervention.
+
+[istio-install]:https://istio.io/latest/docs/setup/install/
+[istio-quickstart]:https://istio.io/latest/docs/setup/getting-started/
+[istioctl]:https://istio.io/latest/docs/setup/install/istioctl/
+[install-method-faq]:https://istio.io/latest/about/faq/#install-method-selection
+
+### Deploying Istio
+
+The `istioctl` command-line program incorporates the concept of
+[profiles][istio-profiles] which provide high level customization of the Istio
+deployment for specialized purposes. For the purposes of this guide we'll use
+the `demo` profile which is meant for testing and evaluation.
+
+Deploy Istio with the `demo` profile:
+
+{% navtabs %}
+{% navtab Command %}
+
 ```console
-$ kubectl apply -f - <<EOF
-  apiVersion: install.istio.io/v1alpha1
-  kind: IstioOperator
-  metadata:
-    namespace: istio-system
-    name: example-istiocontrolplane
-  spec:
-    profile: demo
-EOF
-istiooperator.install.istio.io/example-istiocontrolplane created
+$ istioctl install --set profile=demo -y
 ```
+
+{% endnavtab %}
+{% navtab Response %}
+
 ```console
-$ kubectl describe istiooperator -n istio-system
-...
-...
-Status:
-  Status:  RECONCILING
-...
-...
+✔ Istio core installed
+✔ Istiod installed
+✔ Egress gateways installed
+✔ Ingress gateways installed
+✔ Installation complete
 ```
 
-Wait until the `kubectl describe istiooperator` command returns `Status: HEALTHY`.
+{% endnavtab %}
+{% endnavtabs %}
+
+The Istio components are now deployed to your cluster in the `istio-system`
+[namespace][k8s-namespaces] and you're ready to deploy Kong and a sample
+application with which to utilize the stack.
```

Comment:
I wanted to clean this up because you aren't in fact quite ready, right? You still need the second namespace. So the instructions as a standalone set (without running through the commands) do not _quite_ hang together nicely. It's not horribly broken, though, and I'd like to do another pass after Vik reviews. (I also want to test for myself, just because :D )

Reply:
Go ahead and make changes here np